templates/docs/examples/patterns/accordion/default-deprecated.html

```diff
@@ -0,0 +1,44 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Accordion - deprecated{% endblock %}
```

Comment:
Seems like we only have the deprecated example for default accordion (not the one with headings).
Would be good to have both covered.

I guess we don't need a separate example, maybe just add second accordion with headings to this one, and rename the file to `deprecated.html`. Just so that Percy has something to screenshot, so we are sure we don't introduce some regressions.